cmd/bucket-replication.go

```diff
@@ -508,7 +508,6 @@ func replicateDelete(ctx context.Context, dobj DeletedObjectReplicationInfo, obj
 		VersionSuspended: globalBucketVersioningSys.Suspended(bucket),
 	})
 	if err != nil && !isErrVersionNotFound(err) { // VersionNotFound would be reported by pool that object version is missing on.
```

Comment:
```suggestion
	if err != nil && !isErrVersionNotFound(err) && !isErrObjectNotFound(err){ // VersionNotFound would be reported by pool that object version is missing on.
```

This is perhaps the correct fix here @allanrogerr we still log for other errors.

Reply:
Thanks - modifying